src/Features/CSharp/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProvider.cs

```diff
@@ -121,6 +121,13 @@ public override bool IsTriggerCharacter(SourceText text, int characterPosition,
 
         private INamedTypeSymbol GetCompletionListType(ITypeSymbol type, INamedTypeSymbol within, Compilation compilation)
         {
+            // PERF: None of the SpecialTypes include <completionlist> tags,
+            // so we don't even need to load the documentation.
```

Comment:
we can add something if we needed to (agree not part of this change), but it would be a big win if we can make completion set faster since it will be one of thing that affects throughput.